crates/core/src/surfnet/svm.rs

```diff
@@ -83,6 +84,9 @@ pub struct SurfnetSvm {
     pub signature_subscriptions: HashMap<Signature, Vec<SignatureSubscriptionData>>,
     pub account_subscriptions: AccountSubscriptionData,
     pub slot_subscriptions: Vec<Sender<SlotInfo>>,
+    pub profile_tag_map: HashMap<String, Vec<UuidOrSignature>>,
+    pub simulated_transaction_profiles: HashMap<Uuid, ProfileResult>,
+    pub executed_transaction_profiles: HashMap<Signature, ProfileResult>,
     pub tagged_profiling_results: HashMap<String, Vec<ProfileResult>>,
```

Comment:
Shouldn't we be removing `tagged_profiling_results` and retrieving via `profile_tag_map` instead?